CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,33 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * check if financial account is referenced by financial item.
+   *
+   * @param $financialAccountId Integer
```

Comment:
I thought jenkins  would fail this - the var & the var type are the wrong way around